website/docs/cloud-docs/workspaces/dynamic-provider-credentials/gcp-configuration.mdx

```diff
@@ -67,13 +67,32 @@ Once the service account has been created and granted IAM permissions, you will
 ## Configure Terraform Cloud
 You’ll need to set some environment variables in your Terraform Cloud workspace in order to configure Terraform Cloud to authenticate with GCP using dynamic credentials. You can set these as workspace variables, or if you’d like to share one GCP service account across multiple workspaces, you can use a variable set.
 ### Required Environment Variables
-| Variable                            | Value                                                       | Notes                                                                                          |
-|-------------------------------------|-------------------------------------------------------------|------------------------------------------------------------------------------------------------|
-| `TFC_GCP_PROVIDER_AUTH`             | `true`                                                      | Must be present and set to `true`, or Terraform Cloud will not attempt to authenticate to GCP. |
-| `TFC_GCP_RUN_SERVICE_ACCOUNT_EMAIL` | The service account email used when authenticating to GCP.  ||
-| `TFC_GCP_PROJECT_NUMBER`            | The project number where the pool and other resources live. | This is _not_ the project ID and is a separate number.                                         |
-| `TFC_GCP_WORKLOAD_POOL_ID`          | The workload pool ID.                                       ||
-| `TFC_GCP_WORKLOAD_PROVIDER_ID`      | The workload identity provider ID.                          ||
+| Variable                            | Value                                                                          | Notes                                                                                                                     |
+|-------------------------------------|--------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| `TFC_GCP_PROVIDER_AUTH`             | `true`                                                                         | Must be present and set to `true`, or Terraform Cloud will not attempt to use dynamic credentials to authenticate to GCP. |
+| `TFC_GCP_RUN_SERVICE_ACCOUNT_EMAIL` | The service account email Terraform Cloud will use when authenticating to GCP. ||
+
+You must also include information about the GCP Workload Identity Provider that Terraform Cloud will use when authenticating to GCP. You can supply this information in two different ways:
+
+1. By providing one unified variable containing the canonical name of the workload identity provider
+2. By providing the project number, pool ID, and provider ID as separate variables
```

Comment:
Super nitpicky, non-blocking suggestion: Should we add periods to the end of these list items for consistency? The list items [further down](https://github.com/hashicorp/terraform-docs-common/pull/288/files#diff-e5463c5152fd10b23d9a146e7907a83aad0b2cb783526a9cffc484d1dd23c0aaR48) have them.
```suggestion
1. By providing one unified variable containing the canonical name of the workload identity provider.
2. By providing the project number, pool ID, and provider ID as separate variables.
```